support/DevEnvironments/spack.yaml

```diff
@@ -49,7 +49,7 @@ spack:
   # Charm++:
   # - The 'multicore' backend runs with shared memory on a single node. On
   #   clusters you should choose one of the multi-node backends instead.
-  - 'charmpp@6.10.2: backend=multicore'
```

Comment:
@kidder if your question referred to this line: the colon in the version string means "or newer", so this has been building v7.0.0 all along.

Reply:
yes, that was the line I was asking about; so I am assuming changing the version string to 7.0.0 is the correct thing to do for this PR

Reply:
yep 👍